airsonic-main/src/test/resources/application.properties

```diff
@@ -1,5 +1,8 @@
-spring.mvc.view.prefix: /WEB-INF/jsp/
-spring.mvc.view.suffix: .jsp
+#spring.mvc.view.prefix: /WEB-INF/jsp/
+#spring.mvc.view.suffix: .jsp
+#spring.mvc.pathmatch.use-suffix-pattern=true
+spring.main.allow-bean-definition-overriding=true
```

Comment:
This is turned off by default in Spring Boot 2 and tests error out without it currently (we should fx the behavior of expecting test beans to override regular beans and turn this off again)